.travis.yml

```diff
@@ -56,7 +56,19 @@ jobs:
         # override the default install for language:go
         - true
       script:
-        - ./run-checks --spread
+        - ./run-checks --spread-ubuntu
+    - stage: integration
+      name: spread on other systems than ubuntu
```

Comment:
And this could be named `Fedora, CentOS, OpenSUSE, Debian, Arch`

Reply:
thanks for reviewing! It is ok for me, lets discuss the names on the standup